src/views/settings/NeverAfkFilterSettings.vue

```diff
@@ -0,0 +1,35 @@
+<template lang="pug">
+div
+  div.d-sm-flex.justify-content-between
+    div
+      h5.mb-2.mb-sm-0 Never Treat as Afk Filter
+    div
+      b-form-input(size="sm" v-model="NeverAfkFilterSettings")
```

Comment:
Can we add some validation here? Like, don't allow empty or only-whitespace strings (`s.trim().length != 0`) which would otherwise match all/many events accidentally.

Reply:
I didnt want to just modify the setting onChange(), that would require user to click somewhere outside the `input` element to get the setting updated. But yes added some validation.